src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,7 +1,26 @@
-// Don't forget to import the React library
+import './UserInfo.scss';
+import React from 'react';
+import { User } from '../../react-app-env';
 
-// Create a `UserInfo` component accepting a `user` object and use it to render
-// a `todo.user` in the list with some styling. (Show at least a `name` and an
-// `email` of the `user`)
+type Props = {
+  user: User | null | undefined,
+};
```

Comment:
Я не совсем понимаю, зачем здесь undefined?
Мне кажется, что достаточно просто прописать: 
```suggestion
type Props = {
  user: User,
};
```

Reply:
You don't need `undefined` here